sdk/custom/global/actions.intent.MAIN.yaml

```diff
@@ -1,8 +1,3 @@
 handler:
-  staticPrompt:
-    candidates:
-    - promptResponse:
-        firstSimple:
-          variants:
-          - speech: Start building your action by defining main invocation.
-transitionToScene: actions.scene.END_CONVERSATION
+  webhookHandler: welcome
```

Comment:
The indents in the yaml files seem like they're 2 spaces vs 4 in the js files. We should probably decide on 2 spaces vs 4 so our code is consistent among different files and between the three of us!